client/components/post-schedule/index.jsx

```diff
@@ -168,7 +169,8 @@ class PostSchedule extends Component {
 
 	render() {
 		return (
-			<div className="post-schedule" >
+			<div className="post-schedule">
+				{ this.props.site && <QuerySiteSettings siteId={ this.props.site.ID } /> }
```

Comment:
👍 Let's at least leave a note that Clock is using this. If we looked by at the file by itself it looks like it's querying data for no reason.